r/adbcsnowflake/tools/download-go-vendor-archive.R

```diff
@@ -15,20 +15,14 @@
 # specific language governing permissions and limitations
 # under the License.
 
-PKG_CPPFLAGS=-I$(CURDIR) -DADBC_EXPORT=""
-PKG_LIBS=-L$(CURDIR)/go -ladbc_driver_snowflake
+uri <- Sys.getenv("R_ADBCSNOWFLAKE_VENDORED_DEPENDENCY_URI", "")
 
-CGO_CC = `"${R_HOME}/bin${R_ARCH_BIN}/R.exe" CMD config CC`
-CGO_CXX = `"${R_HOME}/bin${R_ARCH_BIN}/R.exe" CMD config CXX`
-CGO_CFLAGS = $(ALL_CPPFLAGS)
-GO_BIN = $(CURDIR)/go/tmp/go/bin/go.exe
+if (identical(uri, "")) {
+  # When there is a home for this that is not a local file:// URI,
+  # this is where we would set the default.
```

Comment:
aka the asf artifactory? Do we need to setup anything for that in the release scripts?